src/openrct2-ui/windows/Options.cpp

```diff
@@ -1318,6 +1325,22 @@ static void window_options_mousedown(rct_window* w, rct_widgetindex widgetIndex,
                     window_options_show_dropdown(w, widget, AUTOSAVE_NEVER + 1);
                     dropdown_set_checked(gConfigGeneral.autosave_frequency, true);
                     break;
+                case WIDX_AUTOSAVE_AMOUNT_UP:
+                    gConfigGeneral.autosave_amount += 1;
+                    config_save_default();
+                    gfx_invalidate_screen();
```

Comment:
You can remove all the stuff between `config_save_default()` and `break`. Same applies for the other button.

Reply:
Should probably invalidate the widget though, so the current value is displayed properly.